src/components/presets/index.js

```diff
@@ -1,7 +1,8 @@
 import { SelectControl } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import styles from './editor.lazy.scss';
-import React, { useLayoutEffect, useEffect, useState, useRef } from 'react';
+import { useLayoutEffect, useEffect, useState, useRef,memo } from '@wordpress/element';
```

Comment:
```suggestion
import { useLayoutEffect, useEffect, useState, useRef, memo } from '@wordpress/element';
```
avoid adding extra line